### PR TITLE
KBC-2370 test new conditions for token for triggers

### PR DESCRIPTION
**Pull request by jirkasemmler**
Jira: KBC-2370
connection: https://github.com/keboola/connection/pull/3624

Before asking for review make sure that:

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---

### CODE REVIEW

File: tests/Common/TriggersTest.php [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
 namespace Keboola\Test\Common;
 
+use Exception;
 use Keboola\StorageApi\ClientException;
 use Keboola\StorageApi\Options\TokenAbstractOptions;
 use Keboola\StorageApi\Options\TokenCreateOptions;
 use Keboola\Test\StorageApiTestCase;
 
+/**
+ * @retryAttempts 0
```
Comment (martinjunger) [inline tests/Common/TriggersTest.php:14]: proc je tady 0, tyhle testy nepadaji?

**Review by martinjunger — APPROVED** [review]
Vypada to OK. Jeste bezi testy...
... testy OK.

File: tests/Common/TriggersTest.php (hunk 1 above)
Comment (jirkasemmler) [reply tests/Common/TriggersTest.php:14]: sorry, to tady nema byt

Comment (martinjunger) [reply tests/Common/TriggersTest.php:14]: jo, vlastne, to tu mas kvuli lokalu...

**Comment by jirkasemmler** [comment]
hmm, jeste bych to mel pridat do apiary... ale zena vola k obedu, udelam po obede

**Review by zajca — APPROVED** [review]

**Pull request merged**